packages/frontapp/src/app/models/helperFunctions/converTime.ts

```diff
@@ -1,30 +1,19 @@
 import { t } from 'i18next';
 import { StatusTypes } from '@mimir/global-types';
 
-export const periodOfKeeping = Number(
-  process.env['NX_PERIOD_OF_KEEPING'] || 30
-);
-
-export const getDates = (date: Date) => {
-  const currentDate = new Date();
-  const startDate = new Date(date);
-  const returnDate = new Date(
-    startDate.setDate(startDate.getDate() + periodOfKeeping)
-  );
-  return {
-    currentDate,
-    startDate,
-    returnDate,
-  };
-};
-
-export const isOverdue = (date: Date) =>
-  getDates(date).currentDate >= getDates(date).returnDate;
-
-export const getStatus = (status: string | undefined | null, date: any) => {
-  if (!status) return null;
-  if (status === StatusTypes.BUSY)
-    return !isOverdue(date) ? StatusTypes.BUSY : StatusTypes.OVERDUE;
+export const isOverdue = (returnDate?: string) =>
+  returnDate && new Date().getTime() >= new Date(returnDate).getTime();
+
+export const getStatus = (
+  status: string | undefined | null,
+  returnDate?: string
+) => {
+  if (!status) {
+    return null;
+  }
+  if (status === StatusTypes.BUSY) {
+    return !isOverdue(returnDate) ? StatusTypes.BUSY : StatusTypes.OVERDUE;
+  }
```

Comment:
can we later move checking if status is overdue to the backend?

Reply:
Need to think about it. It will be outside of the scope of this task